### ISSUE TITLE
Feedback on paradigms

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#567]
There has been some feedback that the paradigms in new itwêwina are more difficult to parse, in contrast to the old version of itwêwina, cf.

>" guess I find not being able to look up verb charts makes it a little hard to follow. Can you make it so you can see verbs the old way? I found it hard to see the paradigms. All the books are different than how these charts are arranged so I was hunting around quite a bit. This is probably not place to be rearranging conjugates verbs. Maybe do that in a textbook or grammar book instead of here. I thought  the former arrangement was the best feature of the original version. I probably won’t use this version if the charts are all mixed up as it will take longer to find things as they are all mixed up. Maybe allow an option to use it this way or the old verb charts. "

I don't know what the above exactly means. One solution would be to allow for the explicit selection of paradigm types as well as the terminology that is used, so that users can directly select the linguistic paradigm with their choice of terminology. Another would be making the row/column labels more distinct, taking some inspiration from old itwêwina.

**Comment by eddieantonio** [comment]
Closing this in favour of #860/#854
